bin/start-local.js

```diff
@@ -2,20 +2,31 @@
 var spawn = require('child_process').spawn,
     stdout = '',
     execJob,
+    os = require('os'),
     path = require('path'),
-    env = {cwd: path.join(__dirname, '..'), NODE_ENV: process.env.NODE_ENV},
+    yaml = require('js-yaml'),
+    fs = require('fs'),
+    env = {cwd: path.join(__dirname, '..'), NODE_ENV: process.env.NODE_ENV, PATH: process.env.PATH},
     workerJob = null,
     gmeConfig = require(__dirname + '/../config');
 
+const DEEPFORGE_CONDA_ENV =
+    yaml.safeLoad(fs.readFileSync(path.join(__dirname, '..', 'base-environment.yml'))).name;
```

Comment:
I will rename it to `environment.yml` but calling it `base-environment.yml` makes more sense because this file will be updated many times -> creating new files in the future (on installing worker dependencies as well as extension dependencies.)